package rtidb.blobproxy;

option cc_generic_services = true;

message HttpRequest { };
message HttpResponse { };

service BlobProxy {
    rpc Get(HttpRequest) returns(HttpResponse);
}
